.eslintrc.json

```diff
@@ -15,6 +15,7 @@
     "no-shadow": "warn",
     "import/extensions": ["error", "ignorePackages"],
     "no-underscore-dangle": "off",
-    "jest/no-try-expect": "off"
+    "jest/no-try-expect": "off",
+    "no-loop-func": "off"
```

Comment:
Can we please get a rationale of why we’d remove this rule globally and why it’s appropriate in this particular case? https://eslint.org/docs/rules/no-loop-func

Reply:
Yeah, where was this biting you @qgolsteyn?